### ISSUE TITLE
[Script Lab] [High Contrast]: User unable to see the Script tab item button after select HTML tab items.

### ISSUE THREAD

**Issue by lindalu-MSFT** [OfficeDev/office-js#5493]
This **Sev2** bug was opened against the Agaves Add-ins team during an accessibility test pass. 

### Bug Report

Test Environment:
OS name: Microsoft Windows 11 Enterprise.
OS Version: 10.0.22631 Build 22631.
Word Version: Version 2403(Build 17425.20058).
AT: Contrast theme -Desert.
Pre-Requisites:  Settings->Accessibility ->Contrast Themes ->Desert.

#### Expected behavior:

User able to see the Script tab item button after select HTML tab items.

#### Actual behavior:

User unable to see the Script tab item button after select HTML tab items.

#### Steps to Reproduce:

[1.](url)  Apply Pre-Requisite.
2.  Open Word Application and open any document.
3.  Navigate to **Home > Add-ins** and open the **More Add-ins** pane.
4.  Navigate to Office **Store** and **Add** Script Lab add-in.
5.  Navigate to the Script Lab from **My add-ins**.
6.  Navigate main menu items.  
7.  Navigate to the **Script** tab.
8.  Navigate right arrow key to access HTML tab items.
9.  Navigate again left for Script tab items.
10.  Observe the issue.

#### Video repro steps attached
[402aacca-7759-482b-965e-a42ad524150c.webm](https://github.com/OfficeDev/office-js/assets/30682803/c678ab56-bcac-4d59-82ab-9a55a5c0ebb2)
:

**Comment by wandyezj** [comment]
We should move to Fluent 2 controls to avoid this issue.

**Comment by lindalu-MSFT** [comment]
@wandyezj Hey Joe, do you know of anyone who can take on migrating Script Lab to Fluent 2? Since Script Lab is in Maintenance mode, we will have to resolve this as a won't fix otherwise.

**Comment by ElizabethSamuel-MSFT** [comment]
Transferring this issue to the office-js repo, the new home of Script Lab issues.

**Comment by wandyezj** [comment]
I tried this recently and it appears to work.
